README.md

```diff
@@ -1,10 +1,19 @@
-# Kube-OVN charm
+# Kube-OVN Charm
 
-This is an experimental proof-of-concept charm for using Kube-OVN with
-Charmed Kubernetes. This is a subordinate machine charm that relates to
-kubernetes-control-plane and kubernetes-worker via the kubernetes-cni
+Kube-OVN is a CNI implementation based on OVN that provides a rich
```

Comment:
should we maybe include a link to the kube-ovn homepage or overview as well? Maybe https://kubeovn.github.io/docs/v1.10.x/en/

Reply:
Good call, thanks